app/views/devise/registrations/edit.html.erb

```diff
@@ -31,6 +46,16 @@
     <%= f.password_field :current_password, autocomplete: "off" %>
```

Comment:
Think this should be labeled "old password" as this verbiage is more user friendly